flink-runtime/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaRpcService.java

```diff
@@ -116,6 +116,7 @@ public AkkaRpcService(final ActorSystem actorSystem, final Time timeout) {
 			maximumFramesize = actorSystem.settings().config().getBytes(MAXIMUM_FRAME_SIZE_PATH);
 		} else {
 			// only local communication
+			// TODO : this is not a reasonable default.
```

Comment:
Now that we know that this branch is for local communication this _is_ a reasonable default since we never go through netty.